docs/ftldns/cache_dump.md

```diff
@@ -0,0 +1,198 @@
+## Cache dump interpretation
+
+The `dnsmasq` core embedded into `pihole-FTL` prints a dump of the current cache content into the mail log file (default location `/var/log/pihole.log`) when receiving `SIGUSR1`, e.g. by
+
+``` bash
+sudo killall -USR1 pihole-FTL
+```
+
+Such a crash dump looks like
+
+``` plain
+cache size 10000, 0/20984 cache insertions re-used unexpired cache entries.
+queries forwarded 10247, queries answered locally 14713
+queries for authoritative zones 0
+pool memory in use 22272, max 24048, allocated 480000
+server 127.0.0.1#5353: queries sent 10801, retried or failed 69
+server 192.168.2.1#53: queries sent 388, retried or failed 3
+
+Host                                     Address                        Flags      Expires
```

Comment:
Another comment: The misplacement of the header to the lines is intended. I will fix it with a patch later when Simon is fully back (https://github.com/pi-hole/dnsmasq/pull/4).